channels_redis/core.py

```diff
@@ -310,33 +315,34 @@ def _setup_encryption(self, symmetric_encryption_keys):
             # Discard old channels based on group_expiry
             await connection.zremrangebyscore(key, min=0, max=int(time.time()) - self.group_expiry)
 
-            # Return current lot
             channel_names = [x.decode("utf8") for x in await connection.zrange(key, 0, -1)]
 
-        connection_to_channels, channel_to_message, channel_to_capacity, channel_to_key = \
+        channel_keys, connection_to_channel_keys, channel_keys_to_message, channel_keys_to_capacity = \
             self._map_channel_to_connection(channel_names, message)
 
-        for connection_index, channel_redis_keys in connection_to_channels.items():
+        for connection_index, channel_redis_keys in connection_to_channel_keys.items():
+
             # Create a LUA script specific for this connection.
             # Make sure to use the message specific to this channel, it is
             # stored in channel_to_message dict and contains the
             # __asgi_channel__ key.
+
             group_send_lua = """
-                for i=1,#KEYS do
-                    if redis.call('LLEN', KEYS[i]) < tonumber(ARGV[i + #KEYS]) then
-                        redis.call('RPUSH', KEYS[i], ARGV[i])
-                        redis.call('EXPIRE', KEYS[i], %d)
-                    end
-                end
-            """ % self.expiry
+                        for i=1,#KEYS do
+                            if redis.call('LLEN', KEYS[i]) < tonumber(ARGV[i + #KEYS]) then
+                                redis.call('RPUSH', KEYS[i], ARGV[i])
+                                redis.call('EXPIRE', KEYS[i], %d)
+                            end
+                        end
+                    """ % self.expiry
```

Comment:
Why did you indent this without changing it?

Reply:
i guess the IDE did it. i will revert it.